group.go

```diff
@@ -204,6 +208,17 @@ func (c *Client) UpdateGroupManagedBy(dn string, managedBy string) error {
 	return c.Conn.Modify(req)
 }
 
+// UpdateGroupDisplayName updates ldap group displayName
+func (c *Client) UpdateGroupDisplayName(dn string, displayName string) error {
+
+	req := ldap.NewModifyRequest(dn, []ldap.Control{})
+
+	req.Replace("displayName", []string{displayName})
```

Comment:
Here we need to test if the displayName string is empty and emits a delete request instead of the replace, to correctly handle the case where the displayName attribute is already set and we want to remove it. There is the same logic on the description attribute here: https://github.com/Ouest-France/goldap/blob/4539515cdce09ae136d2a23f32e12ae0556b7f23/group.go#L160-L164

Reply:
Thanks, done